Remove Outdated Target/Alias Usage from Guides (#1354)
zeit/docs: pages/guides/deploying-a-mongodb-powered-api-with-node-and-now.mdx

With Now, you can deploy your app to [different environments](/docs/v2/custom-domains#deploying-with-your-domain), depending on what stage your app is at; either staging or production.

To deploy your MongoDB API, use the following [Now CLI](/download) command from your terminal:

<Snippet dark text="now" />
<Caption>Deploying with Now CLI in one command.</Caption>

> For [production deployments](/docs/v2/custom-domains#deploying-with-your-domain), you can deploy to a production domain of your choice. To do so, [add a custom domain](https://zeit.co/docs/v2/custom-domains/#adding-a-domain) to your project.

Reviewer: We're saying "production" twice here and it's a bit tricky to read.
Maybe:
```suggestion
To use your domain for [production deployments](/docs/v2/custom-domains#deploying-with-your-domain), [add a custom domain](https://zeit.co/docs/v2/custom-domains/#adding-a-domain) to your project.
```
